### ISSUE TITLE
EJP XML manifest - deliver author emails to the right post-editorial processes

### ISSUE THREAD

**Issue by Melissa37** [elifesciences/package-ejp-raw-output-zip#3]
## Problem / Motivation
WHO: Production/Bot/Production Vendor
WHEN: Between Export from EJP/xPub and delivery to production vendor
WHERE: In eLife bot processes
WHAT: All author emails available in EJP
WHY: EJP exports all emails for authors, but not all of them are used in publication, but all are used in Bot processes

## Proposed solution
- Reference other tickets in this repo for the whole process, this is a specific task in the conversion process.
- The editorial process collects all author emails, but we only publish emails for corresponding authors.
- The eLife bot collects all author emails and we send every author an email when their article is published.

Example of XML output from EJP:
```<contrib-group><contrib contrib-type="author" corresp="yes" id="author-24343"><name><surname>Cembrowski</surname><given-names>Mark S</given-names></name><ext-link ext-link-type="orcid">http://orcid.org/0000-0001-8275-7362</ext-link><aff><addr-line><named-content content-type="department"></named-content></addr-line><institution>Janelia Research Campus, Howard Hughes Medical Institute</institution><addr-line><named-content content-type="city">Ashburn</named-content></addr-line><country>United States</country><email>cembrowskim@janelia.hhmi.org</email></aff></contrib><contrib contrib-type="author" id="author-51569"><name><surname>Wang</surname><given-names>Lihua</given-names></name><aff><addr-line><named-content content-type="department"></named-content></addr-line><institution>Janelia Research Campus, Howard Hughes Medical Institute</institution><addr-line><named-content content-type="city">Ashburn</named-content></addr-line><country>United States</country><email>wangl10@janelia.hhmi.org</email></aff></contrib>```

Linked to:
```<custom-meta><meta-name>Corresponding author for proofs</meta-name><meta-value>author-24343</meta-value></custom-meta>```

## Assumptions
- At one time we requested the production vendor retain all author emails and if proofs are not responded to after a given time, an automated email went to all authors. However, because some author lists are so long, we decided not to go ahead with this proposal and introduce a manual step for this eventuality, when production staff will select who to send the proof to if the corresponding author is unresponsive.
Therefore, the production vendor does not need this data and so we should not send it to them.
The author emails sent out by eLife on publication does require all of these emails.

- Any author with ```corresp="yes"``` in their ```<contrib>``` must have their ```<email>``` delivered within their ```<contrib>``` to the production vendor

## Clarification needed 

- @gnott to confirm where all the author emails are gathered from for his process for author emails. Could this be a good place to extract them from instead of current processes?
>The author publication email system gets the email addresses from a CSV file produced by the EJP query tool.
I'm not sure collecting them from these manifest files would be an improvement. There will always be a need to query all the authors from a central place, and if the CSV file method will change, I would suggest a new issue outside of this project to discuss where to store the data and how it will be compiled.
- Is the ```<custom-meta>``` tagging the best way for the production vendor to be told who to send the proof email to?
- Is the production vendor going to be responsible for sending out the proofing email? If not, we should not send them this data.

---

## Tasks
@gnott to fill out

## Technical notes
@gnott to fill out

## User interface / Wireframes
NA

@gnott @eLifeProduction

**Comment by gnott** [comment]
> @gnott to confirm where all the author emails are gathered from for his process for author emails. Could this be a good place to extract them from instead of current processes?

The author publication email system gets the email addresses from a `CSV` file produced by the EJP query tool.

I'm not sure collecting them from these manifest files would be an improvement. There will always be a need to query all the authors from a central place, and if the CSV file method will change, I would suggest a new issue outside of this project to discuss where to store the data and how it will be compiled.

**Comment by Melissa37** [comment]
Thanks @gnott I've added that response into the opening part @Maelplaine is this what I should be doing as owner of the ticket to ensure the scoping part is up to date?

Thanks!

**Comment by Maelplaine** [comment]
Yes @Melissa37 that's useful to have the answer in the ticket description because it can get lost in the comments threads after a while.
